libuavcan_drivers/stm32/driver/src/internal.hpp

```diff
@@ -122,6 +134,22 @@ struct CriticalSectionLocker
     }
 };
 
+#elif UAVCAN_STM32_FREERTOS
+
+struct CriticalSectionLocker
+{
+    //TODO napisati funkcije
```

Comment:
English please. ;)

What function do you want to write here? Everything seems to be in place, am I missing something?